src/components/NavBar.css

```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter&family=Nosifer&family=Roboto:wght@100&family=UnifrakturMaguntia&display=swap');
+
+.navbar {
+  height: 3rem;
+  margin: 0 0 2rem;
+  padding: 1.4rem 4rem 1.7rem 4.3rem;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid #121212;
+  align-items: center;
+
+  /* box-shadow: 0 4px 20px -3px rgba(0, 0, 0, 0.71); */
```

Comment:
- [ ] If you don't need commented lines of codes please remove it from your project.